scripts/backup

```diff
@@ -6,42 +6,39 @@
 
 source /usr/share/yunohost/helpers
 
-#=================================================
-# DECLARE DATA AND CONF FILES TO BACKUP
-#=================================================
-ynh_print_info --message="Declaring files to be backed up..."
+ynh_print_info "Declaring files to be backed up..."
 
 #=================================================
 # BACKUP THE APP MAIN DIR
 #=================================================
 
-ynh_backup --src_path="$install_dir"
+ynh_backup "$install_dir"
 
 #=================================================
 # BACKUP THE APP DATA
 #=================================================
 
-ynh_backup --src_path="$data_dir" --is_big
+ynh_backup "$data_dir"
```

Comment:
Does that mean the data will still be excluded from light backups?

Reply:
yes, the "--is-big" mechanism is now handled by the core, automatically for $data_dir, maybe also the log directory etc